doc/whats-new.md

```diff
@@ -1,144 +1,195 @@
+CSV export format of the ground motion fields which is
+now the same both for event based and scenario calculators. Also the
+internal storage of the GMFs has changed to make the import/export
+procedure simpler.
+
+We implemented risk statistics for the `classical_damage` calculator
+and the `classical_bcr` calculator: before they were missing and the
+user had to do the computation manually from the individual realizations.
+
+We implemented the concept of *asset tag* in the exposure. The
+risk calculators are now able to compute losses aggregated by tag.
+For a definition of the *tag* concept please see the current version
+of the manual.
+
+We changed the algorithm used in the calculation of loss curves (both
+aggregated and per-asset curves). Now the losses are given in terms of
+return periods. If the return periods are not specified in the `job.ini`
+file, the engine automatically generates a sensible range of return
+periods. There is not need to specify the `loss_ratios` in the `job.ini`
+file as it was required by the previous algorithm.
+
+The statistical loss curves exporter for `classical_risk` calculations
+was exporting bogus numbers, even if the data in the datastore were
+correct. This has been fixed. The statistical loss curves exporter for
+`event_based_risk` was not implemented: now it is.
+
+The generation of loss_maps and curves in event based risk has been optimized:
+now it is done in parallel whenever possible.
+
+There is a better error message when running a risk file in absence of a
+preceding hazard calculation.
+
+WebUI and QGIS plugin
 ---------------------
 
-- The engine was calling the routine computing the statistics even when
-  not needed. This was inefficient and has been fixed.
-- The error checking when parsing source models in format NRML 0.5 has been
-  improved: now in case of error one gets the name of the incorrect node and
-  its line number, just like for NRML 0.4.
-- There is now a clear error message if the user does not set the
-  `calculation_mode` in the `job.ini` file.
-- We improved the error message when the rupture mesh spacing is too small.
-- We added a new `.npz` exporter for the output `dmg_by_asset_npz`.
-- There is a new `.csv` exporter for the aggregate loss curves, replacing
-  the deprecated XML exporter.
-- Some preliminary work for the Python 3 installers has been done.
+We are visualizing the `calculation_mode` field in the WebUI now,
+instead of the `job_type` field, which is less specific.
+
+We removed an excessive check from the WebUI: now if an output exists,
+it can be downloaded even if the calculation was not successful.
+
+Now we show the user the error message in the case of a calculation
+that cannot be deleted from the WebUI.
+
+We changed the .npz exporters of the hazard outputs to make the import
+into a single QGIS layer easier.
+
+We have now in place an automatic testing mechanism making sure that
```

Comment:
I would change this into something like:
"We have now in place an automatic integration testing mechanism making sure that all the outputs of the demos that are loadable by the QGIS plugin are actually imported and visualized correctly."